src/test/java/org/example/integration/AuthIntegrationTests.java

```diff
@@ -0,0 +1,56 @@
+package org.example.integration;
+
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+
+import org.example.TestApplication;
+import org.example.TestConfiguration;
+import org.example.models.LoginRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+import java.util.Base64;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthIntegrationTests {
+    public static final DropwizardAppExtension<TestConfiguration> APP =
+            new DropwizardAppExtension<>(TestApplication.class);
+
+    @Test
+    void login_shouldLogin_whenCorrectCredentials(){
+        Client client = APP.client();
+        LoginRequest login = new LoginRequest("admin", "admin");
```

Comment:
again, shouldn't have valid login creds in code, you can pass these in as env vars

Reply:
Created env vars for user and pass